From: Antonio Terceiro <asa@terceiro.xyz>
Date: Thu, 13 Feb 2020 08:15:05 -0300
Subject: mentos.py: break on EOF

This makes it a lot easier to test the script by providing it pre-cooked
inputs.
---
 lib/pygments/mentos.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/lib/pygments/mentos.py b/lib/pygments/mentos.py
index 002a6cb..7f4e475 100755
--- a/lib/pygments/mentos.py
+++ b/lib/pygments/mentos.py
@@ -267,6 +267,9 @@ class Mentos(object):
             # our JSON header.
             size = sys.stdin.read(32)
 
+            if not size:
+                break
+
             lock.acquire()
 
             try:
